libr/core/cconfig.c

```diff
@@ -2010,6 +2010,7 @@ R_API int r_core_config_init(RCore *core) {
 	SETPREF ("asm.marks", "true", "Show marks before the disassembly");
 	SETPREF ("asm.cmtrefs", "false", "Show flag and comments from refs in disasm");
 	SETPREF ("asm.cmtpatch", "false", "Show patch comments in disasm");
+	SETPREF ("asm.showpayloads", "false", "Show payload bytes in disasm");
```

Comment:
maybe asm.payloads is enough, the "show" thing is implicity in all the subvars of asm.